packages/ChangesReloaded-Tests.package/CRBrowserTest.class/instance/testLoadSelectedChanges.st

```diff
@@ -0,0 +1,6 @@
+testing
+testLoadSelectedChanges
+
+	| changeRecord |
+	changeRecord := ChangeRecord new
+		
```

Comment:
... wow. So this is a placeholder test then? 😅

Reply:
This is a deleted test, now :D